build.cake

```diff
@@ -73,12 +84,21 @@ Task("ReplaceStubWithExample")
 Task("TestUsingExampleImplementation")
     .IsDependentOn("ReplaceStubWithExample")
     .Does(() => {
-        var allProjects = GetFiles(buildDir + "/*/*.csproj");
+        var allProjects = GetFiles(exercisesBuildDir + "/*/*.csproj");
         Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
     });
 
+Task("BuildGeneratorsSolution")
+    .IsDependentOn("CopyGenerators")
+    .Does(() => {
+        var generatorsProject = GetFiles(generatorsBuildDir + "/*.csproj");
+        Parallel.ForEach(generatorsProject, parallelOptions, (project) => NuGetRestore(project.FullPath));
+        Parallel.ForEach(generatorsProject, parallelOptions, (project) => DotNetBuild(project.FullPath));
```

Comment:
This should be DotNetCoreBuild as this is a .NET Core 2 project.

Reply:
The generators solution contains one project, so we shouldn't need to run this build step in parallel.